roles/kvm/tasks/systemd.yml

```diff
@@ -0,0 +1,11 @@
+---
+##
+# Configure IOMMU support for systemd-boot
+##
+
+- name: Configure IOMMU for systemd-boot
+  become: true
+  ansible.builtin.command: kernelstub --add-options "{{ iommu_param }}"
+  register: kernelstub_result
+  changed_when: kernelstub_result.rc == 0
```

Comment:
Nitpick: Also, all these `changed_when` with `rc == 0` can be simplified with just `changed_when: true` (and the the `failed_when` with `rc != 0` can be omitted, as discussed before)